website/docs/d/repositories.html.markdown

```diff
@@ -27,6 +27,8 @@ The following arguments are supported:
 
 * `query` - (Required) Search query. See [documentation for the search syntax](https://help.github.com/articles/understanding-the-search-syntax/).
 
+* `sort` - (Optional) Sorts query. Valid values include `stars`, `fork`, and `updated`. Defaults to `updated`.
```

Comment:
Would this sound better as: 
```suggestion
* `sort` - (Optional) Sorts the repositories returned by the specified attribute. Valid values include `stars`, `fork`, and `updated`. Defaults to `updated`.
```

Reply:
Agreed. Was having a time with the wording on that.